Exercise_A/index.html

```diff
@@ -17,29 +16,195 @@
+                <p class="bullshit-photos-div-overlay-title text-center fw-bolder">FOUR THING</p>
+                <p class="bullshit-photos-div-overylay-text text-center">Bullshit</p>
+              </div>
+          </div>
+      </div>
+      <div class="bullshit-text d-flex flex-column align-items-center justify-content-center mx-auto">
+          <h3 class="bullshit-text-title fw-normal text-center text-white">Seriously, It's This Fucking Template !</h3>
+          <a href="#page-top"><button type="button" class="white-button btn btn-light fw-bolder animate__animated animate__tada wow my-4">THAT'S AMAZING</button></a>
+      </div> 
+    </section>
     <!-- contact me container -->
+    <section id="call-me">
+        <div class="call-me-div">  
+          <div class="call-me-text container">
+            <h3 class="call-me-information text-center">
+              My Contact Information
+            </h3>
+            <hr id="my-contact-hr">
+            <p class="my-contact-para text-center">
+              Here's my phone number and email address, because hey that's part of the template and why would you not include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in from all the companies wanting my insane website design skills.
+            </p>
+            <p class="my-contact-para text-center">Honestly this template does look really nice, though.</p>
+          </div>
+          <div class="contact-icon container row mx-auto">
+            <div class="contact-icon-content d-flex flex-column align-items-center col-lg-6">
+              <i class="fas fa-phone-alt animate__animated animate__bounceIn wow"></i>
+              <p class="my-contact-para text-center">+1-608-317-1888</p>
+            </div>
+            <div class="contact-icon-card d-flex flex-column align-items-center col-lg-6">
+              <i class="far fa-envelope animate__animated animate__bounceIn wow"></i>
+              <a class="email-link my-contact-para text-center" href="mailto:shermanxcc@gmail.com">holy@shit.com</a>
+            </div>
+          </div>
+        </div>
+    </section>
 
     <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
-
     <!-- 
       All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
-     -->
+    -->
+    <!-- CDN Boostrap.JS -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-U1DAWAznBHeqEIlVSCgzq+c9gqGAJn5c/t99JyeKa9xxaYpSvHU5awsuZVVFIhvj" crossorigin="anonymous"></script>
+    <!-- WOW.JS -->
+    <script src="./assets/js/wow.js"></script>
+              <script>
+              new WOW().init();
+              </script>
+    <!-- Custom JS -->
+    <script src="./assets/js/script.js"></script>
   </body>
 </html>

```

Comment:
Bravo! You have nailed the cloning of the website. It shows that you have excellent knowledge of HTML, CSS, Bootstrap, DOM, and JavaScript. Keep up the good work, and keep on grinding.